conversion/tts_tokenizer_profiles.py

```
@@ -0,0 +1,164 @@
+    if str(cfg.get("nemo_version", "")) != TOKENIZER_PROFILE_NEMO_VERSIONS[profile]:
+        raise ValueError(f"unsupported {profile} nemo_version: {cfg.get('nemo_version')!r}")
+    for name, target in TOKENIZER_TARGETS[profile].items():
+        if tokenizers[name].get("_target_") != target:
+            raise ValueError(
+                f"unsupported {profile} tokenizer target for {name}: "
+                f"{tokenizers[name].get('_target_')!r}"
+            )
+    expected_japanese_case = "upper" if profile == "v2602" else "lower"
+    japanese_case = tokenizers["japanese_phoneme"].get("g2p", {}).get("ascii_letter_case")
+    if japanese_case != expected_japanese_case:
+        raise ValueError(f"unsupported {profile} Japanese ascii_letter_case: {japanese_case!r}")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)